package.json

```diff
@@ -37,9 +37,9 @@
     "sense": "nebula sense",
     "spec": "scriptappy-from-jsdoc -c ./spec-configs/props.conf.js",
     "start": "nebula serve",
+    "test:e2e": "playwright test",
```

Comment:
the rendering tests became e2e tests?

Reply:
The default for nebula charts is e2e tests https://github.com/qlik-oss/nebula.js/blob/master/commands/create/templates/sn/none/_package.json#L22

We have not really figured out if we need to split them up or not in the script. I think both rendering and integration tests can both live under the test:e2e command

Reply:
ok makes sense